### DISCUSSION TITLE
Support for MeanderingProgrammer/render-markdown.nvim

### DISCUSSION THREAD

**Discussion by phuongpd2k** [olimorris/onedarkpro.nvim#256] [Q&A]
I’m currently using Obsidian and rendering markdown with the render-markdown.nvim plugin, but I’ve noticed that OneDark doesn’t fully support it. Could you consider making some updates in the future? :(

Onedark theme:
<img width="842" alt="image" src="https://github.com/user-attachments/assets/89cf4d8a-d735-4eaa-ac89-e744e24086ca">

Tokyonight theme:
<img width="833" alt="image" src="https://github.com/user-attachments/assets/6d286df4-bc9e-495c-a3b8-37c24f294ff0">

**Comment by olimorris** [comment]
Could you add a PR for this? The Contributing.md outlines how to do it.

**Reply by phuongpd2k** [reply]
Yes, I’ve made some updates for this, but I’m just using a fixed color. Is that okay?

**Reply by olimorris** [reply]
What do you mean "fixed colors"? We should be using the colors from the theme's palette, like in the other plugins that the theme supports.

**Comment by olimorris** [comment]
I've added support for this in the latest release 👍🏼
